### ISSUE TITLE
Feature Request: Ability to run tests in parallel

### ISSUE THREAD

**Issue by BustyLoli-Chan** [karma-runner/karma#2695]
As a developer working on an application with an ever growing test suite it would be really nice if we possessed the ability to run any particular suite of tests in parallel with any other suite as they almost never conflict. 

I've noticed that there is a concurrency flag that is set to `infinity` by default, but I never have more than a single browser pop up and open. Also, despite all attempts to change this number, I never notice any behavioral difference in the test runner itself.

If this feature isn't available I think it would be incredibly helpful to have, and if this is a feature that is currently available, please forgive me and teach me how to capitalize upon this feature.

**Comment by maksimr** [comment]
Duplicates of https://github.com/karma-runner/karma/issues/439
